deploy/openshift/sws.yaml

```diff
@@ -101,6 +101,7 @@ rules:
   attributeRestrictions: null
   resources:
   - routerules
+  - destinationpolicies
```

Comment:
That is not part of the review, but a comment. What is this apiGroups and resources for? Thank you

Reply:
The object "ClusterRole" defines access rights that can be assigned (in our case) to a service account linked with the SWS application.
To get access to the API of Kubernetes and Istio, we need to specifically define this rights in a similar object that is attached with a "ClusterRoleBinding".

If you look, there are two main rules for apiGroups, the detault one, used for kubernetes objects, and the "config.istio.io" one, which is used for Istio. Istio API is an extension of Kubernetes, (like OpenShift one).